Cesium3DTilesSelection/include/Cesium3DTilesSelection/RasterOverlayDetails.h

```diff
@@ -0,0 +1,29 @@
+#pragma once
+
+#include <CesiumGeometry/Rectangle.h>
+#include <CesiumGeospatial/BoundingRegion.h>
+#include <CesiumGeospatial/Projection.h>
+
+#include <vector>
+
+namespace Cesium3DTilesSelection {
+struct RasterOverlayDetails {
+  RasterOverlayDetails();
+
+  RasterOverlayDetails(
```

Comment:
I updated the doc in the new commit now. The constructor is needed in the function [GltfUtilities::createRasterOverlayTextureCoordinates](https://github.com/CesiumGS/cesium-native/blob/1fe78d990800e08a0cb39b70fa49e71c1e1fdd67/Cesium3DTilesSelection/src/GltfUtilities.cpp#L308). Since we define default constructor, that constructor won't be generated by the compiler